src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/use-travel-search-filters-state.tsx

```diff
@@ -17,7 +17,7 @@ type TravelSearchFiltersState =
       anyFiltersApplied: boolean;
       resetTransportModes: () => void;
       disableFlexibleTransport: () => void;
-      closeRef: React.Ref<any>;
+      onCloseFocusRef: React.RefObject<any> | undefined;
```

Comment:
I think this doesn't need to be optional.

Reply:
No longer optional 👍